index.html

```diff
@@ -310,6 +319,224 @@ <h2>Contact Me</h2>
+                      <img src="./images/Counter.png" alt="">
+                    <p class="p1">FULL STACK DEV</p>
+                      <img src="./images/Counter.png" alt="">
+                    <p class="p1">2015</p>
+                  </div>
+                  <p class="primary2">
+                    Experimental content creation feature that allows users to<br /> add to an existing story over the course of a day without<br /> spamming their friends.
+                  </p>
+                  <ul>
+                    <li><a href="#">html</a></li>
+                    <li><a href="#">Ruby on rails</a></li>
+                    <li><a href="#">css</a></li>
+                    <li><a href="#">javascript</a></li>
+                  </ul>
+          <div class="action1">
+          <a href="#"><p>See Project</p></a>
+          </div>
+        </div>
+      </div>
+      <div class="card-work3">
+        <div class="card-image3">
+          <!-- <img src="./images/d3.png" alt="card-work"> -->
+        </div>
+        <h2 class="h2">FACEBOOK 360</h2>
+      <div class="left2-block">
+          <div class="frame3">
+            <p>FACEBOOK</p>
+            <img src="./images/Counter.jpg" alt="">
+            <p class="p1"> FULL STACK Dev</p>
+            <img src="./images/Counter.jpg" alt="">
+            <p class="p1">2015</p>
+          </div>
+          
+            <p class="primary3">
+              Exploring the future of media in Facebook's first Virtual Reality <br />app; a place to discover and enjoy 360 photos and videos on <br />Gear VR.
+          </p>
+          <ul>
+            <li><a href="#">html</a></li>
+            <li><a href="#">Ruby on rails</a></li>
+            <li><a href="#">css</a></li>
+            <li><a href="#">javascript</a></li>
+          </ul>
+          <div class="action2">
+            <a href="#"><p>See Project</p></a>
+          </div>
+          </div>
+      </div>  
+      <div class="card-work4">
+        <div class="card-image4">
+          <img src="./images/d5.png" alt="card-work">
+        </div>
+        <div class="left1-block">
+          <h2 class="h2">
+            Uber Navigation
+          </h2>
+          <div class="frame4">
+            <p>FACEBOOK</p>
+              <img src="./images/Counter.png" alt="">
+            <p class="p1">FULL STACK DEV</p>
+              <img src="./images/Counter.png" alt="">
+            <p class="p1">2015</p>
+          </div>
+          <p class="primary2">
+            Experimental content creation feature that allows users to<br /> add to an existing story over the course of a day without<br /> spamming their friends.
+          </p>
+          <ul>
+            <li><a href="#">html</a></li>
+            <li><a href="#">Ruby on rails</a></li>
+            <li><a href="#">css</a></li>
+            <li><a href="#">javascript</a></li>
+          </ul>
+  <div class="action1">
+  <a href="#"><p>See Project</p></a>
+  </div>
```

Comment:
- [ ] This `See Project` button should take you to the Live Demo of the project or to the GitHub repository. So please implement it